cfgov/regulations3k/models/django.py

```diff
@@ -174,6 +174,8 @@ def numeric_label(self):
         part, number = self.label.split('-')[:2]
         if number.isdigit():
             return '\xa7\xa0{}.{}'.format(part, number)
+        else:
+            return ''
```

Comment:
How would you feel about returning a [`sortable_label()`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/django.py#L13) instead? It converts alphabetical labels into their positional numeric equivalent.

Reply:
√